src/Security/Guard/JWTTokenAuthenticator.php

```diff
@@ -57,7 +57,7 @@ public function onAuthenticationFailure(Request $request, AuthenticationExceptio
         $response = parent::onAuthenticationFailure($request, $authException); // TODO: Change the autogenerated stub
 
         if ($authException instanceof InvalidTokenException || $authException instanceof ExpiredTokenException) {
-            $response->headers->clearCookie('BEARER');
+            $response->headers->clearCookie('BEARER', '/', null, false, true, 'none');
```

Comment:
Does this also remove the cookie if samesite is set to another value than `none`?

Reply:
removing a cookie is established by setting a cookie which is expired. the `clearCookie` method here does exactly that (see below)

so 'overwriting' the BEARER does not care about the existing cookie and its samesite settings 

![image](https://user-images.githubusercontent.com/13519286/124125572-10580000-da7a-11eb-9bfb-a109433310bd.png)

Reply:
But I mean, if there is a cookie with samesite set to strict or lax, will setting an expired cookie with samesite set to none clear it?

Reply:
a cookie will replace a previous cookie if the name, path and domain are the same. samesite is not a matching field.

alternatively we can use the same $samesite config input here as used in the SuccessHandler when creating the cookie